fem/datacollection.cpp

```diff
@@ -431,6 +428,27 @@ void VisItDataCollection::RegisterField(const std::string& name,
       }
    }
 
+   DataCollection::RegisterField(name, gf);
+   field_info_map[name] = VisItFieldInfo("nodes", gf->VectorDim(), LOD);
+   visit_levels_of_detail = std::max(visit_levels_of_detail, LOD);
+}
+
+void VisItDataCollection::RegisterQField(const std::string& name,
+                                         QuadratureFunction *qf)
+{
+   int LOD = -1;
+   Mesh *mesh = qf->GetSpace()->GetMesh();
+   for (int e=0; e<qf->GetSpace()->GetNE(); e++)
+   {
+      int locLOD = GlobGeometryRefiner.GetRefinementLevelFromElems(
+                      mesh->GetElementBaseGeometry(e),
+                      qf->GetElementIntRule(e).GetNPoints());
```

Comment:
Why not just use `IntegrationRule::GetOrder` to derive the LOD?

Reply:
There is unfortunately not a (very) clear connection between quadrature order and quadrature points. Moreover not each distribution of quadrature points is associated with a refinement, at least not how the refinements are programmed at this time.  Off course some of this "logic" could also be moved to visit, and LOD could simply be the order (or order +1) similar as for the FESpace,

Reply:
Moreover the quadrature order is ussually ~ twice as high as the order of the shapefunctions.